### PR TITLE
Ollama embedding support

### PR DESCRIPTION
**Pull request by gaya3-vijayakumar**
## What

Support Ollama embedding models

## Why

To be able to run embeddings locally

## How

Add one more adapter for embedding

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions / Env Variables

-

## Notes on Testing

- Add Ollama embedding as an adapter
- Use the same to run indexing and prompts in prompt studio

## Screenshots

image

image

image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by Deepak-Kesavan — APPROVED** [review]
Look good.

File: src/unstract/adapters/embedding/ollama/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# Unstract Ollama Embedding Adapter
```
Comment (muhammad-ali-e) [inline src/unstract/adapters/embedding/ollama/README.md:1]: Just a thought, Atleast we can add some ref docs here. For this, we can add ollama docs you reffered. (Not mandatory now)

Comment (gaya3-vijayakumar) [reply src/unstract/adapters/embedding/ollama/README.md:1]: @muhammad-ali-e I have addressed this comment to some extent. Pls check and approve if that is ok.

File: src/unstract/adapters/embedding/ollama/src/ollama.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+        try:
+            embedding_batch_size = EmbeddingHelper.get_embedding_batch_size(
+                config=self.config
+            )
+            embedding: BaseEmbedding = OllamaEmbedding(
+                model_name=str(self.config.get(Constants.MODEL)),
+                embed_batch_size=embedding_batch_size,
+            )
+            return embedding
+        except Exception as e:
```
Comment (harini-venkataraman) [inline src/unstract/adapters/embedding/ollama/src/ollama.py:57]: @gaya3-zipstack Is there any other exceptions we could handle here?

**Review by harini-venkataraman — APPROVED** [review]
Added a minor comment, otherwise LGTM

File: src/unstract/adapters/embedding/ollama/src/ollama.py (hunk 2 above)
Comment (gaya3-vijayakumar) [reply src/unstract/adapters/embedding/ollama/src/ollama.py:57]: Hard to find from the code base @harini-venkataraman So for now, keeping it in line with other adapters

**Pull request merged**